src/ceph-disk/ceph_disk/main.py

```diff
@@ -207,6 +207,12 @@ def is_dmcrypt(ptype, name):
 
 DEFAULT_FS_TYPE = 'xfs'
 SYSFS = '/sys'
+PROCDIR = '/proc'
+
+FREEBSD = False
+if platform.system() == 'FreeBSD':
+    FREEBSD = True
+    PROCDIR = '/compat/linux/proc'
```

Comment:
nit. could move the
```python
FREEBSD = False
PROCDIR = '/proc'
```
to the "else" branch. so the structure of the code is more balanced and compact.